packages/nc-gui/components/smartsheet/Form.vue

```diff
@@ -730,7 +730,7 @@ watch(view, (nextView) => {
                     <LazySmartsheetDivDataCell class="relative">
                       <LazySmartsheetCell
                         v-model="formState[element.title]"
-                        class="nc-input"
+                        class="nc-input truncate"
                         :class="`nc-form-input-${element.title.replaceAll(' ', '')}`"
```

Comment:
i tried fixing that but looks like the css `nc-form-input-longtext` is over riding my css , i tried making my css !important but not working , what should i do ? i tried searching for property `nc-form-input-longtext` but couldnt find anywhere